### ISSUE TITLE
Set value of DayName variable to current day in enterTimeTable.dart

### ISSUE THREAD

**Issue by nityanandagohain** [nityanandagohain/in-time#56]
The dayName variable should be initialized to current day.

**Comment by piedcipher** [comment]
I would like to work on this issue. I'm a GSSoC 19 Participant.
Slack Display Name - tirth[P]

**Comment by nityanandagohain** [comment]
@piedcipher sure go ahead.

**Comment by piedcipher** [comment]
Okay I'm working on this. Also I created this PR #63 for Issue #62. So should I added commits for Issue #56 (this one) in same PR?

**Comment by nityanandagohain** [comment]
No create a new PR.

**Comment by piedcipher** [comment]
Okay. So can you please review my PR #63? I've fixed Issue #62 as well as Issue #56. Once PR #63 is merged I'll create PR for this issue (#56).

**Comment by piedcipher** [comment]
Thanks! That was quick! :smile: Creating a PR

**Comment by nityanandagohain** [comment]
You can create as many PR as you want by creating new branches from the master branch and making changes in them, you don't need to wait for  a PR to be merged.
